config/ompi_setup_mpi_fortran.m4

```diff
@@ -307,6 +307,35 @@ AC_DEFUN([OMPI_SETUP_MPI_FORTRAN],[
     OMPI_FORTRAN_GET_KIND_VALUE([C_INT32_T], 9, [OMPI_FORTRAN_C_INT32_T_KIND])
     OMPI_FORTRAN_GET_KIND_VALUE([C_INT64_T], 18, [OMPI_FORTRAN_C_INT64_T_KIND])
 
+    #
+    # See if "attributes deprecated" is supported
+    # - False positives were seen with compilers like XL which can recognize
+    #   this option by default, but if the user specifies some options to
+    #   the wrapper compiler later it will fail to recognize the option.
+    # - For not just limit this check to compilers that we know will work
```

Comment:
It looks like this "gfortran 11+" comment annotation was not kept.

It would be useful to keep this annotation, because I initially tested the functionality with gfortran 10, and was confused as to why it did not work.